ui/src/dialogs/AuthenticationDialog/AuthenticationDialog.jsx

```diff
@@ -86,12 +90,24 @@ export class AuthenticationDialog extends Component {
 
   onRegisterClick(e) {
     e.preventDefault();
-    this.setState({ firstSection: 'hide', secondSection: '' });
+    this.setState({
+      firstSection: 'hide',
+      secondSection: '',
+      submitted: false,
+    });
   }
 
   onSignInClick(e) {
     e.preventDefault();
-    this.setState({ firstSection: '', secondSection: 'hide' });
+    this.setState({
+      firstSection: '',
+      secondSection: 'hide',
```

Comment:
Could firstSection / sectionSection be handled with a single toggle?

Reply:
Yes, probably, but note that I didn’t actually introduce this code in this PR (it’s just formatted differently), and I think refactoring it wouldn’t be worth it at this time.